scripts/clean_start_dev.sh

```diff
@@ -0,0 +1,12 @@
+execute() {
+    "$@"
+    local status=$?
+    if [ $status -ne 0 ]; then
+        echo "error with $1" >&2
+        exit $status
+    fi
+    return $status
+}
+
+execute find . -type f -name "package.json" -maxdepth 2 -execdir sh -c 'rm -rf node_modules && npm i' \; &&
+execute docker compose build --no-cache && docker compose up -d
```

Comment:
[nit] some shells may not execute the last line in a script if the script file doesn't end in a line break

Reply:
Fixed!